pebblo/app/utils/utils.py

```diff
@@ -203,3 +205,50 @@ def get_pebblo_server_version():
 
     __version__ = importlib.metadata.version("pebblo")
     return __version__
+
+
+def acquire_lock(lock_file_path: str):
+    """
+    Acquire lock using given lock file. It will be busy wait, polling after every 2 seconds.
+    """
+    sleep_time = 2
```

Comment:
Why the 2-second sleep? Is the lock expected to be held for a long time?

Reply:
not long time as such, but we need wait if lock is already acquired.